generated-platform-project/quarkus-camel/bom/pom.xml

```diff
@@ -216,7 +216,7 @@
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-storage-blob-batch</artifactId>
-        <version>12.10.0</version>
+        <version>12.8.0</version>
```

Comment:
@aloubyansky I was a bit surprised by this change?

Reply:
I noticed that too. I'll need to look into that. The only azure artifact in core is functions java library. I'll have a look.